README.md

```diff
@@ -455,7 +455,7 @@ per section is not supported.
 
 Note on tests: as seen in the above example, it is recommended to wrap them all
 in `<![CDATA[..]]>` tags.  This avoids XML parsing errors when special
-characters are encoutered, such as the `&` used to output to stderr in bash.
+characters are encoutered, such as the `>&2` used to output to stderr in bash.
```

Comment:
Still missing a fix for the "encoutered" typo.

Reply:
Ug, forgot about that one.  Fixed and squashed in https://github.com/hastexo/hastexo-xblock/pull/89/commits/1c8b6eab8a50e8bb8b5c53be7dc538c1328cce52.